src/librustdoc/formats/cache.rs

```diff
@@ -516,6 +519,31 @@ impl<'a, 'tcx> DocFolder for CacheBuilder<'a, 'tcx> {
     }
 }
 
+#[derive(Default)]
+pub struct LocalResources {
+    /// The key is the original location of the resource. The value is the new name.
+    pub(crate) resources_to_copy: FxHashMap<PathBuf, String>,
+    /// This will be used when generating the HTML, once everything is generated, we copy these
+    /// files into the static folder.
```

Comment:
We shouldn't put per-crate files into `static.files/`, which is used for toolchain-specific static files. If we go this route, there should be a separate directory for invocation-specific files.

Reply:
Absolutely. The RFC mentions that there won't be a crate folder. ;)